### DISCUSSION TITLE
如何将该懒人配置与 Scoop 安装的 MPV 系统相结合

### DISCUSSION THREAD

**Discussion by SamuNatsu** [hooke007/mpv_PlayKit#498] [Q&A 问答]
rt，翻来覆去看批处理不知道怎么把这个 portable_config 和 scoop mpv 的 portable_config 合并，直接覆盖就能用吗

**Comment by hooke007** [comment]
我不知道你的具体用途。
我也不知道为什么scoop的mpv会强制带portable_config。
portable_config就是应该放在mpv的同级路径下。

**Reply by SamuNatsu** [reply]
其实我想问的是能否直接合并 portable_config 文件夹，还是说 install 批处理里面还做了其他会造成破坏性修改的东西

**Reply by hooke007** [reply]
不谈来源，从来没有合并的说法。都是删旧的留新的

> install 批处理里面还做了其他会造成破坏性修改的东西

没有

**Comment by izen000** [comment]
直接把portable_config里的文件放进scoop_mpv的portable_config文件夹不就行了吗？
